content/applications/finance/accounting/bank/feeds/bank_statements.rst

```diff
@@ -33,30 +33,19 @@ the :guilabel:`Bank` window, or click on the drop-down menu (:guilabel:`⋮`) bu
 
 Next, select the file you want to import and click on :guilabel:`Upload`.
```

Comment:
I don't see the "upload" option. I would just say "Next, select and upload the file you want".